### ISSUE TITLE
Assign var

### ISSUE THREAD

**Issue by JesusKB** [bobthecow/mustache.php#297]
I need a assign var function so much! I see this - https://github.com/mardix/ThickMustache, but is old.
My framework can't work without assign var correctly. Please, give me a example how to add this feature.

**Comment by val4o0o0** [comment]
Yes, this would be nice. Maybe assign_var_array is nice thing too. (Working like method arrayiterator)

**Comment by bobthecow** [comment]
Variable assignment in templates doesn't align with the Mustache design philosophy, so it's unlikely we'll ever implement this. You can often accomplish similar things using "blocks" (via template inheritance) or "filters" (e.g. to add defaults or derived values)
